### ISSUE TITLE
Communicate MeiliSearch error to instantSearch

### ISSUE THREAD

**Issue by bidoubiwa** [meilisearch/instant-meilisearch#381]
In case of error, MeiliSearch sends it usual body to instantSearch. Resulting to this: 

```
instantsearch.js@4:2 Uncaught (in promise) Error: Uncaught, unspecified "error" event. ([object Object])
```

We should either stop the return process to instant search or find how to communicate it properly to instantsearch

**Comment by EricPaulson** [comment]
Seconded. The lack of info this message is providing me while trying to get this set up is costing me a significant amount of time.

**Comment by bidoubiwa** [comment]
Fixed by #216
